README.md

```diff
@@ -1 +1,45 @@
-# fish_species
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] On your readme, kindly please add the section "license" according to the [Microverse readme template](https://github.com/microverseinc/readme-template).  The link featured in that section should point to a file at the root of your project (the license file).   Just copy it from the mentioned template:

![image](https://user-images.githubusercontent.com/57421823/168004669-c3026da8-8102-4a87-bb49-f7a5a9a6abbf.png)

That way, we will meet this requirement, and your project will be well documented 💪

Reply:
Thank @Meltrust  for your suggestions, now it's addressed. :handshake: